pkg/koordlet/qosmanager/framework/config.go

```diff
@@ -21,26 +21,34 @@ import (
 )
 
 type Config struct {
-	ReconcileIntervalSeconds   int
-	CPUSuppressIntervalSeconds int
-	CPUEvictIntervalSeconds    int
-	MemoryEvictIntervalSeconds int
-	MemoryEvictCoolTimeSeconds int
-	CPUEvictCoolTimeSeconds    int
-	OnlyEvictByAPI             bool
-	QOSExtensionCfg            *QOSExtensionConfig
+	ReconcileIntervalSeconds    int
+	CPUSuppressIntervalSeconds  int
+	CPUEvictIntervalSeconds     int
+	MemoryEvictIntervalSeconds  int
+	MemoryEvictCoolTimeSeconds  int
+	CPUEvictCoolTimeSeconds     int
+	OnlyEvictByAPI              bool
+	EvictByCopilotAgent         bool
+	EvictByCopilotEndPoint      string
+	EvictByCopilotPodLabelKey   string
+	EvictByCopilotPodLabelValue string
+	QOSExtensionCfg             *QOSExtensionConfig
 }
 
 func NewDefaultConfig() *Config {
 	return &Config{
-		ReconcileIntervalSeconds:   1,
-		CPUSuppressIntervalSeconds: 1,
-		CPUEvictIntervalSeconds:    1,
-		MemoryEvictIntervalSeconds: 1,
-		MemoryEvictCoolTimeSeconds: 4,
-		CPUEvictCoolTimeSeconds:    20,
-		OnlyEvictByAPI:             false,
-		QOSExtensionCfg:            &QOSExtensionConfig{FeatureGates: map[string]bool{}},
+		ReconcileIntervalSeconds:    1,
+		CPUSuppressIntervalSeconds:  1,
+		CPUEvictIntervalSeconds:     1,
+		MemoryEvictIntervalSeconds:  1,
+		MemoryEvictCoolTimeSeconds:  4,
+		CPUEvictCoolTimeSeconds:     20,
+		OnlyEvictByAPI:              false,
+		EvictByCopilotAgent:         false,
+		EvictByCopilotEndPoint:      "/var/run/yarn-copilot/yarn-copilot.sock",
```

Comment:
The koordlet manages the directory path of the host `/var/run` as `sysutil.Conf.VarRunRootDir`. So it is recommended to concatenate the path with the sysutil.Conf.